Documentation/getting-started-ubuntu-ppa.md

```diff
@@ -22,6 +22,7 @@ $ sudo apt install nnstreamer
 * nnstreamer-dev : Required to build C function-filters and to build your own nnstreamer plugins.
 * nnstreamer-edgetpu : Allows to use edge-TPU in a pipeline.
 * nnstreamer-flatbuf : Allows to convert-from and decode-to flatbuf streams.
+* nnstreamer-misc: Allows to use join plugin.
```

Comment:
Only ` join` is included now in `nnstreamer-misc` package, but other plugins may be added in the future.
How about changing like this?
```suggestion
* nnstreamer-misc: Provides additional gstreamer plugins for nnstreamer pipelines. Included plugins: join
```